cmd/d2/main.go

```diff
@@ -127,19 +127,34 @@ func run(ctx context.Context, ms *xmain.State) (err error) {
 	}
 	ms.Log.Debug.Printf("using layout plugin %s (%s)", *layoutFlag, pluginLocation)
 
+	var pw png.Playwright
+	if filepath.Ext(outputPath) == ".png" {
+		pw, err = png.InitPlaywright()
+		if err != nil {
+			return err
+		}
+		defer func() error {
+			err = pw.Cleanup()
+			if err != nil {
+				return err
+			}
+			return nil
```

Comment:
again, returning an error doesn't do anything. u have a named return parameter, called `err`. if that is nil, then take the result of Cleanup() and set it to that. if it's not nil, then just ignore, there's another error anyway.

Reply:
Ohhhhhhhh